src/site/content/en/fast/priority-hints/index.md

```diff
@@ -576,12 +577,15 @@ You can also enable the **Experimental Web Platform Features** flag in Chrome to
 
 The following is a comparison between a set of images and a script file loaded with different priorities. Note that the default priorities for images and scripts are applied on Chrome 94. On the other hand, the priority set through the importance attribute is applied when tested on Chrome 96.
 
-Following is the relevant mark-up included in the HTML file.
+Specifying an origin trial token in an HTML `<meta>` element is often the quickest and most convenient method. For Priority Hints to work during the origin trial, however, you'll need to specify your token in an `Origin-Trial` HTTP response header:
```

Comment:
I'd add a qualifier along the lines of: for Priority Hints to work for resources loaded very early in the page, the origin trial token needs to be specified in the HTTP response header.

The complication here is that the original trial _does_ work, it just doesn't apply to the files whose load was initiated by the preload scanner. Files loaded later one _will_ honor the importance attribute.

Reply:
I think, if we can make it clear that, if you use a `<meta>` tag, the origin trial gets initialized too late to apply to some of the earliest resources loaded, that's fairly clear what's going on.

Reply:
I pushed an update to the copy @philipwalton.